templates/profiles/change_password.html

```diff
@@ -0,0 +1,20 @@
+{% extends "base.html" %}
+{% load i18n %}
+{% load markdownify %}
+{% load courses_extra %}
```

Comment:
No need to load `markdownify` and `courses_extra` as they are not used.

Besides `courses_extra` is in different app, we do not want to make cross reference, unless necessary.

Reply:
done, sorry about that, forget to remove it after copy and paste from different file